postgis/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM timescale/timescaledb:0.0.11-beta
+FROM timescale/timescaledb:latest
```

Comment:
This is a bit weird since it might not match the ENV variable in the Docker file, leading to inconsistent builds.